apps/www/registry/stories/separator.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { Separator } from "@/registry/default/ui/separator"
+
+/**
+ * Visually or semantically separates content.
+ */
+const meta = {
+  title: "ui/Separator",
+  component: Separator,
+  tags: ["autodocs"],
+  argTypes: {},
+} satisfies Meta<typeof Separator>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+/**
+ * The default form of the separator.
+ */
+export const Horizontal: Story = {
+  render: () => (
+    <div className="flex gap-2">
+      <div>Left</div>
+      <Separator orientation="vertical" className="h-auto" />
```

Comment:
these two stories are backwards. this is a vertical separator and is marked as horizontal, and vice versa

Reply:
Maybe this is a semantic thing, but when I look at [the stories](https://65cbbef5e6055247b3744fab-kvmbrnnava.chromatic.com/?path=/docs/ui-separator--docs) having them named inverse to the orientation makes more sense to me.  _Horizontally_ separated items have a _vertical_ oriented `Seperator` 🤷‍♂️

Reply:
The doc block says this is the default form of the separator, however `orientation="horizontal"` will be the default. 

Regardless, thanks for this PR even if it never goes in. I'm using it as a reference for adding stories to my codebase.